### PR TITLE
Resolves COLDBOX-770

### PR DESCRIPTION
**Pull request by jclausen**

### CODE REVIEW

File: system/web/context/RequestContext.cfc [hunk 1]
```diff
@@ -1241,7 +1241,7 @@ component serializable=false accessors="true"{
 		}
 
 		//  Set content headers
```
Comment (lmajano) [inline system/web/context/RequestContext.cfc]: Can you add the following check 
```
// detect if extension is already included in name argument, if not add
		if ( listlast( arguments.name,"." ) != arguments.extension ) {
			arguments.name = arguments.name =& ".#arguments.extension#";
		}
```

**Comment by elpete** [comment]
Wasn't there already a PR for this?

https://github.com/ColdBox/coldbox-platform/pull/395

**Comment by lmajano** [comment]
> Wasn't there already a PR for this?
> 
> #395

Yes, I am trying to consolidate them now.

**Comment by jclausen** [comment]
@jclausen : Updated with requested changes

File: system/web/context/RequestContext.cfc (hunk 1 above)
Comment (jclausen) [reply system/web/context/RequestContext.cfc]: Done. See above -  used an append instead of a reset of the variable

**Pull request merged**